policies.tex

```diff
@@ -47,7 +47,7 @@ \subsection{}After electing all other candidates, Members-at-Large will be elect
 \subarticle{Appointments}
 
 \subsection{}The Executive Committee may appoint additional officers, including but not limited to those required by the constitution or standing policies.
-\subsection{}The term of an appointed officer shall not extend beyond the end of the term of the appointing EC.
+\subsection{}The term of an appointed officer shall not extend more than one month beyond the end of the term of the appointing EC, and shall lapse upon their replacement taking office.
```

Comment:
This should be "and shall lapse upon their replacement's taking office", although that reads weirdly.  Perhaps reword to "and shall lapse when their replacement takes office."

Although regardless of the wording above, the two parts of the sentence sound somewhat contradictory.  Perhaps "The term of an appointed officer shall lapse when their replacement takes office, no later than one month beyond the end of the term of the original appointing EC."

Still not quite happy with that text.